### DISCUSSION TITLE
HORI - Battle Pad Luigi not correctly detected

### DISCUSSION THREAD

**Discussion by RigleGit** [cathery/sys-con#157] [Troubleshooting]
**Description**
Probably it's not detected as Pro device but another one, so some controls are switched (B is acting as A and viceversa)

**How to reproduce**
1. In the menu, press A
2. You'll get B answer
3. Same with other buttons

- Your Switch Firmware version: 9.2.0
- Your version of Atmosphere: 0.10.5
- Your version of sys-con: 0.5.3

**Comment by Dacrenzio** [comment]
I have a temporary solution: the HORI battlepad is detected as a XBOX 360 controller. If you write 
KEY_FACE_DOWN = FACE_RIGHT
KEY_FACE_RIGHT = FACE_DOWN
KEY_FACE_LEFT = FACE_UP
KEY_FACE_UP = FACE_LEFT

in the configuration file.ini of the xbox 360. This will fix the A, B, X and Y button. You need to reset the switch in order to get the changes and obl the Xbox 360 will have the buttons swapped after this

**Comment by neutral8l** [comment]
> **Description** Probably it's not detected as Pro device but another one, so some controls are switched (B is acting as A and viceversa)
> 
> **How to reproduce**
> 
> 1. In the menu, press A
> 2. You'll get B answer
> 3. Same with other buttons
> 
> * Your Switch Firmware version: 9.2.0
> * Your version of Atmosphere: 0.10.5
> * Your version of sys-con: 0.5.3

Did you manage to fix this?
